### PR TITLE
update some cluster information

### PR DESCRIPTION
**Pull request by wdpypere**

### CODE REVIEW

File: intro-HPC/ch_connecting.tex [hunk 1]
```diff
@@ -370,8 +370,8 @@ \section{First Time connection to the HPC infrastructure}
    cluster - full - free -  part - total - running - queued
              nodes  nodes   free   nodes    jobs      jobs
 -------------------------------------------------------------------
-    golett    168      2     16     200      N/A       N/A
-    phanpy     15      0      1      16      N/A       N/A
+     doduo    168      2     16     200      N/A       N/A
```
Comment (stdweird) [inline intro-HPC/ch_connecting.tex:373]: kirlia?

File: intro-HPC/ch_running_batch_jobs.tex [hunk 2]
```diff
@@ -353,8 +353,8 @@ \subsection{Search for modules}
 %\ifgent{}%
     You will need to load all module(s) on any one of the lines below before the "example/1.2.3" module is available to load.
 
-      cluster/golett
-      cluster/phanpy
+      cluster/kirlia
+      cluster/joltik
```
Comment (stdweird) [inline intro-HPC/ch_running_batch_jobs.tex:357]: doduo?

File: intro-HPC/ch_running_batch_jobs.tex [hunk 3]
```diff
@@ -823,7 +823,7 @@ \subsection{Specifying the cluster on which to run}
 \begin{prompt}
 %\shellcmd{module avail cluster/}%
 ------------------------------------------------------------------------------------ /etc/modulefiles/vsc ------------------------------------------------------------------------------------
-   cluster/golett (S)    cluster/phanpy (S)    cluster/skitty (S)    cluster/swalot (S)    cluster/victini (S,L)
+   cluster/doduo (S)    cluster/kirlia (S)    cluster/skitty (S)    cluster/swalot (S)    cluster/victini (S,L)
```
Comment (stdweird) [inline intro-HPC/ch_running_batch_jobs.tex:826]: kirlia?

File: intro-HPC/ch_running_batch_jobs.tex (hunk 2 above)
Comment (wdpypere) [reply intro-HPC/ch_running_batch_jobs.tex:357]: ah, you want it exact, I thought they were just examples

File: intro-HPC/ch_running_batch_jobs.tex (hunk 3 above)
Comment (boegel) [reply intro-HPC/ch_running_batch_jobs.tex:826]: kirlia is there too ;)

Comment (wdpypere) [reply intro-HPC/ch_running_batch_jobs.tex:826]: kirlia is right there

File: intro-HPC/ch_connecting.tex (hunk 1 above)
Comment (boegel) [reply intro-HPC/ch_connecting.tex:373]: Order is also different in the actual output (old to new), we should fix that too...

Comment (wdpypere) [reply intro-HPC/ch_connecting.tex]: fixed

File: intro-HPC/ch_running_batch_jobs.tex (hunk 2 above)
Comment (boegel) [reply intro-HPC/ch_running_batch_jobs.tex:357]: It is just an example, but we usually have all software installed on all clusters, so they should be all listed here (except the hidden ones).

```
      cluster/doduo
      cluster/joltik
      cluster/kirlia
      cluster/skitty
      cluster/swalot
      cluster/victini
```

File: intro-HPC/ch_running_batch_jobs.tex [hunk 4]
```diff
@@ -556,7 +556,7 @@ \subsection{Getting cluster state}
 
 
 \begin{prompt}
-%\shellcmd{module swap cluster/phanpy}%
+%\shellcmd{module swap cluster/kirlia}%
```
Comment (boegel) [inline intro-HPC/ch_running_batch_jobs.tex:560]: If you change this, also change the output below...

Comment (wdpypere) [reply intro-HPC/ch_running_batch_jobs.tex:560]: done

**Comment by wdpypere** [comment]
jeesh, these tests take ages :)

File: intro-HPC/ch_running_batch_jobs.tex [hunk 5]
```diff
@@ -353,11 +353,12 @@ \subsection{Search for modules}
 %\ifgent{}%
     You will need to load all module(s) on any one of the lines below before the "example/1.2.3" module is available to load.
 
-      cluster/golett
-      cluster/phanpy
+      cluster/kirlia
+      cluster/joltik
```
Comment (boegel) [inline intro-HPC/ch_running_batch_jobs.tex:357]: Actual output has these in alphabetical order, so `doduo` on top, then `joltik`, `kirlia`, ...

**Review by boegel — CHANGES_REQUESTED** [review]

File: intro-HPC/ch_running_batch_jobs.tex (hunk 5 above)
Comment (wdpypere) [reply intro-HPC/ch_running_batch_jobs.tex:357]: I don't think `module spider example/1.2.3` has actual output :)

Comment (boegel) [reply intro-HPC/ch_running_batch_jobs.tex:357]: For *any* `module spider` command, the clusters will be in alphabetical order, so please fix.

Comment (wdpypere) [reply intro-HPC/ch_running_batch_jobs.tex:357]: which module spider command do you want me to use?

**Review by boegel — APPROVED** [review]
lgtm

File: sites/gent/macros.tex [hunk 6]
```diff
@@ -63,7 +63,7 @@
 \macro{\hpcinfo}{hpc@ugent.be}
 \macro{\hpcusersml}{hpc-users@lists.ugent.be}
 \macro{\hpcannounceml}{hpc-announce@lists.ugent.be}
-\macro{\operatingsystem}{CentOS 7.7 (golett, phanpy, skitty, swalot, victini)}
+\macro{\operatingsystem}{CentOS 7.9 (skitty, swalot, victini, joltik, kirlia) or RHEL 8.2 (doduo)}
```
Comment (itkovian) [inline sites/gent/macros.tex:66]:
```suggestion
\macro{\operatingsystem}{CentOS 7.9 (skitty, swalot, victini, joltik, kirlia) and RHEL 8.2 (doduo)}
```

**Pull request merged**